mono/profiler/decode.c

```diff
@@ -3063,6 +3279,178 @@ dump_heap_shots (void)
 	}
 }
 
+/* This is a very basic escape function that escapes < > and &
+   Ideally we'd use g_markup_escape_string but that function isn't
+	 available in Mono's eglib. This was written without looking at the
```

Comment:
g_markup_escape_string is from the main glib which is LGPL, and eglib is X11 licensed. My understanding is that putting X11 licensed code into LGPL licensed code is permitted, but not the other way round.